### DISCUSSION TITLE
No response from GitHub Support

### DISCUSSION THREAD

**Discussion by jbonusatfortra** [community/community#108229] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey all,
I have been struggling to get in touch with Github support. My ticket has been open for 2 weeks and I have followed up twice. Has anyone else experienced these wait times?

It is blocking something important on my end.

Thank you,
Josh

**Comment by queenofcorgis** [comment, marked as answer]
Hi @jbonusatfortra, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
